server/consumer.go

```diff
@@ -768,6 +782,8 @@ func (mset *stream) addConsumerWithAssignment(config *ConsumerConfig, oname stri
 		// Ephemerals will always have inactive thresholds.
 		// Add in 1 sec of jitter above and beyond the default of 5s.
 		o.dthresh = JsDeleteWaitTimeDefault + time.Duration(rand.Int63n(1000))*time.Millisecond
+		// Only stamp config with default sans jitter.
+		o.cfg.InactiveThreshold = JsDeleteWaitTimeDefault
```

Comment:
Discussing with Alberto, he has a point that if Durable is now deprecated, if the app were to specify only Name and no inactive threshold, I believe that the code will still fall into that "if" statement and make it an ephemeral, no?

Reply:
Correct, you would have to fill in Durable or provide a filter subject.

Durable is not deprecated per se, but previously it was very specific in semantics that it would not go away until removed.

Moving forward we will support both legacy models, but new way is all have a Name and an optional Inactive Threshold and R factor.